bindings/pyroot/pythonizations/python/ROOT/_facade.py

```diff
@@ -258,6 +258,8 @@ def RDF(self):
         ns = self._fallback_getattr('RDF')
         try:
             from libROOTPythonizations import MakeNumpyDataFrame
+            import DistRDF
+            ns.Distributed = DistRDF
```

Comment:
To be decided (related to other comments above):
1. We inject `DistRDF` as `ROOT.RDF.Distributed`
2. We create a new `ROOT.RDF.Distributed`.

Reply:
I think in the end it's best if we have `ROOT.RDF.Distributed` as a separate module